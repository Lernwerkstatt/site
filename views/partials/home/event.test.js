const event = require("./event.hbs");

describe("event.hbs", () => {
  it("should match the snapshot", () => {
    expect(
      event({
        name: "Webkurs",
        date: "13.12.2018",
        time: "18:00-20:00",
        eventlink: "https://www.facebook.com/pg/dielernwerkstatt/events",
        dayicon: "img/calendar/sunday.png"
      })
    ).toMatchSnapshot();
  });
});
