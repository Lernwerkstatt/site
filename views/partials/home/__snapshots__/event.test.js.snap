// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`event.hbs should match the snapshot 1`] = `
"<li class=\\"list-group-item\\">
  <a href=\\"https://www.facebook.com/pg/dielernwerkstatt/events\\" target=\\"_blank\\">
    <p class=\\"h5\\">
      <img src=\\"img/calendar/sunday.png\\" />
      Webkurs
    </p>
    <small class=\\"text-muted\\">13.12.2018 - 18:00-20:00</small>
  </a>
</li>"
`;
